packages/stitch/tests/errors.test.ts

```diff
@@ -1,8 +1,10 @@
-import { graphql } from 'graphql';
+import { graphql, GraphQLError, buildSchema } from 'graphql';
 
 import { makeExecutableSchema } from '@graphql-tools/schema';
 
 import { stitchSchemas } from '../src/stitchSchemas';
+import { ExecutionResult } from '@graphql-tools/utils';
+import { Executor } from 'packages/delegate/src';
```

Comment:
```suggestion
import { Executor } from '@graphql-tools/delegate';
```

P.S.: not a big deal but maybe sort the imports as well

Reply:
Thanks so much!